### ISSUE TITLE
library/EventSource/Types.hs: error:  No instance for (Semigroup Properties)

### ISSUE THREAD

**Issue by juhp** [YoEight/eventsource-api#4]
In Stackage Nightly:

```
Building library for eventsource-api-1.3.0..
[1 of 6] Compiling EventSource.Types ( library/EventSource/Types.hs, dist/build/EventSource/Types.o )

library/EventSource/Types.hs:116:10: error:
    • No instance for (Semigroup Properties)
        arising from the superclasses of an instance declaration
    • In the instance declaration for ‘Monoid Properties’
    |
116 | instance Monoid Properties where
    |          ^^^^^^^^^^^^^^^^^
```

**Comment by YoEight** [comment]
@juhp  Thanks for reporting this, I'm on it!

**Comment by YoEight** [comment]
Now that https://github.com/alphaHeavy/protobuf/pull/36 got merged, this issue will resolve by itself!

**Comment by jkachmar** [comment]
Unsure what the relation is between `Properties` and the `protobuf` stuff, but unfortunately it looks like it's still failing with the same error @juhp reported above 😞

**Comment by YoEight** [comment]
I missed the mark here. It's like I was able to use last `nightly` using GHC 8.2.2, hence why it passed on my machine (tm). On another note, adding an instance of `Semigroup` wouldn't fix the issue and would have still required fixing `protobuf` (used as a transient dependency if you use the `geteventstore` backend).  I'm also patching my `eventstore` lib who had a similar build issue on GHC 8.4.2. I would cut a new release of all those packages in the next hours.

Thanks for your patience 😄

**Comment by YoEight** [comment]
I fixed it with this commit 5a00ccb30bce2bf36d08f23957a7fe8b63877ebd. I also cut `eventsource-api-1.3.1` which is also uploaded on Hackage.
